static/index.html

```diff
@@ -33,14 +34,32 @@ <h3 id="title"></h3>
         let stateURL = new URL(window.location.pathname + "state", window.location.origin)
         let stopURL = new URL(window.location.pathname + "stop", window.location.origin)
         let playButton = document.getElementById("play");
+        let loopButton = document.getElementById("loop");
+        let loopBool = false;
```

Comment:
lets call it this instead
```suggestion
        let videoWillLoop = false;
```

Reply:
done